src/ai/backend/manager/cli/api.py

```diff
@@ -1,19 +1,16 @@
 from __future__ import annotations
 
 import asyncio
-import importlib
 import json
 import logging
 from pathlib import Path
 from typing import TYPE_CHECKING
 
 import aiofiles
-import aiohttp_cors
 import click
 import graphene
-from aiohttp import web
 
-from ai.backend.manager.openapi import generate_openapi
+from ai.backend.manager.openapi import _generate
```

Comment:
Avoid marking friend function as private. In Python, starting a function name with `_` means "private".

Reply:
Please check this. @Yaminyam